src/main.ts

```diff
@@ -19,6 +19,14 @@ export type NextOptions = {
   timeout?: number
 }
 
+export type EmitterOnOptions = {
+  signal?: AbortSignal
+}
+
+export type EmitterOnceOptions = {
+  signal?: AbortSignal
+}
```

Comment:
I really like these names, would like to see `EmitterNextOptions` as well to be consistent

Reply:
I'll add to my todo list. Wanna clean this repo up a bit